### PR TITLE
Update RHOAM add-on info with details about the free entitlement

### PR DESCRIPTION
**Pull request by jonquilwilliams**
JIRA: https://issues.redhat.com/browse/OSDOCS-2403

**Preview:** 

**ROSA -** https://deploy-preview-36014--osdocs.netlify.app/openshift-rosa/latest/adding_service_cluster/rosa-available-services.html#osd-rhoam_rosa-available-services

**OSD -** https://deploy-preview-36014--osdocs.netlify.app/openshift-dedicated/latest/adding_service_cluster/available-services.html

Branch = dedicated-4  No CP and no milestones needed.

### CODE REVIEW

File: modules/aws-cloudwatch.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+[id="aws-cloudwatch_{context}"]
+
+= AWS CLoudWatch
```
Comment (pneedle-rh) [inline modules/aws-cloudwatch.adoc:4]: @jonquilwilliams from a quick search it looks like the product name is "Amazon CloudWatch". You could update this line and the other references to the name in this PR to reflect this.

**Comment by 0kashi** [comment]
I would clarify in the last bullet that it is 100k calls per day.   Also isnt the formatting normally with a lowercase "k"?

Overall looks alright to me (though maybe the product owners of RHOAM should comment too?)

**Comment by yuwang-RH** [comment]
LGTM

**Comment by EricPonvelle** [comment]
Closing this as these changes were merged in this PR https://github.com/openshift/openshift-docs/pull/37941

**Pull request closed without merging**